aspnetcore/razor-components/index.md

```diff
@@ -1,65 +1,37 @@
 ---
-title: Introduction to Razor Components
+title: Introduction to ASP.NET Core Razor Components
 author: guardrex
-description: Explore Blazor, a .NET web framework using C#/Razor and HTML that runs in the browser with WebAssembly.
+description: Explore ASP.NET Core Razor Components, a .NET web framework using C#/Razor and HTML.
 monikerRange: '>= aspnetcore-3.0'
 ms.author: riande
 ms.custom: mvc
-ms.date: 01/29/2019
+ms.date: 02/05/2019
 uid: razor-components/index
 ---
-# Introduction to Razor Components
+# Introduction to ASP.NET Core Razor Components
```

Comment:
This topic was originally written as an intro to Blazor as a client-side SPA framework and was intended for .NET and non-.NET audiences. It doesn't read quite right as an intro to razor components. So I think it needs to restructured a bit to better introduce what razor components (not Blazor) is all about.

The key ideas that I think should come across in this topic are:
- Razor Components are used for client-side web app development with .NET and ASP.NET Core. 
- With Razor Components you can build rich interactive UIs using .NET instead of JavaScript. 
- Razor Components are self-contained chunks of user interface (UI), such as a page, dialog, or form. The can handle user events and define flexible UI rendering logic.
- Razor Components are typically authored using Razor syntax, a natural blend of HTML and C#.
- Razor Component are designed to efficiently update the UI by rendering into an in-memory representation of the UI that can be efficiently diff'd with what was previously rendered. Only the parts of the UI that changed need to be updated. 
- Razor Components are just .NET classes that get built into .NET assemblies and can then be shared and distributed as NuGet packages.
- Razor Components can call an browser API or JS library through JS interop
- Razor Components are similar to Razor Pages and MVC Views in that they both use Razor. But unlike pages and views, which are built around a request/reply model, components are used specifically for handling UI composition.
- Razor Components can be used with your existing Razor Pages and MVC apps. You can add Razor Components to your views and pages.
- Razor Components can be hosted client-side in the browser (Blazor), or server-side in an ASP.NET Code app, where UI updates are handled over a SignalR connection.
- Razor Components support a variety of features: parameters, event handling, data binding, routing, dependency injection, layouts, templating, cascading values, etc.